devicetypes/smartthings/zwave-battery-thermostat.src/zwave-battery-thermostat.groovy

```diff
@@ -12,20 +12,23 @@
  *
  */
 metadata {
-	definition (name: "Z-Wave Battery Thermostat", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.thermostat", genericHandler: "Z-Wave") {
+	definition (name: "Z-Wave Battery Thermostat", namespace: "smartthings", author: "SmartThings",
+			ocfDeviceType: "oic.d.thermostat", genericHandler: "Z-Wave", runLocally: false,
+			executeCommandsLocally: true, minHubCoreVersion: '000.033.0001') {
 		capability "Actuator"
 		capability "Temperature Measurement"
 		capability "Thermostat Heating Setpoint"
 		capability "Thermostat Cooling Setpoint"
 		capability "Thermostat Operating State"
 		capability "Thermostat Mode"
 		capability "Thermostat Fan Mode"
+		capability "Humidity Measurement"
```

Comment:
@dkirker do you know a vid I could use that supports all of the existing capabilities + humidity?

Reply:
Isn't the capability supposed to be `Relative Humidity Measurement` ? (missing Relative)

Reply:
@prjct92eh2 yes, thanks